src/Components/Dot/index.jsx

```
@@ -1,6 +1,6 @@
 import PropTypes from "prop-types";
 
-const Dot = ({ color = "gray", size = "4px" }) => {
+const Dot = ({ color = "gray", size = "4px", style }) => {
```

ESLint

[error] 'style' is missing in props validation

(react/prop-types)